client/src/ui/molecules/PeoplePickerSelector.tsx

```diff
@@ -82,14 +82,31 @@ const PeoplePickerSelector = ({
                             setSearchTerm(event.currentTarget.value);
                         }}
                     />
-                    <span className="typography__body typography__body--primary people-picker__guidance">
+                    <span className="typography__body--primary people-picker__guidance">
                         {min
                             ? `${t('ui:validation--peoplepicker_guidance-prefix')} ${min} ${t(
                                   'ui:validation--peoplepicker_guidance-suffix',
                               )}`
                             : null}
                     </span>
                 </div>
+                <div className="people-picker__selected-tabs">
+                    {
+                        locallySelected.map(selectedPersonId => {
```

Comment:
Can this not be simplified with a `people.filter` ?

Reply:
We'll loose order by filtering over the People array. We should ask what desired order these should be in. By selection? Alphabetically?